src/style.css

```diff
@@ -1 +1,73 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.form:not(:last-child) {
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.searchbar__container {
+  position: relative;
+}
+
+.searchbar {
+  width: 100%;
+  background-image: url('./images/Search.svg');
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's good that you have 'Arial, sans-serif' as fallbacks, but ensure they are correctly ordered according to their similarity to the primary font.